lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -110,8 +110,8 @@ function dosomething_user_menu_alter(&$items) {
   $items['user/%'] = array(
     'page callback' => 'dosomething_user_get_view',
     'page arguments' => array(1),
-    'access callback' => 'user_access',
-    'access arguments' => array('access user profiles'),
+    'access callback' => 'user_view_access',
```

Comment:
@angaither okay yeah info is a path that uses [these permissions](https://github.com/DoSomething/phoenix/blob/ef62285b87f37c914cc5a35a6a6fdf0efca94bd1/lib/modules/dosomething/dosomething_user/dosomething_user.features.user_permission.inc#L345) and I don't think anything we did should mess with that or /edit?